src/main/java/frc/robot/subsystems/intake/BallIntakeIO.kt

```diff
@@ -0,0 +1,89 @@
+package frc.robot.subsystems.intake
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.CANSparkMaxLowLevel
+import com.revrobotics.SparkMaxAbsoluteEncoder
+import edu.wpi.first.math.geometry.Rotation2d
+import edu.wpi.first.math.util.Units
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+
+interface IntakeIO  {
+
+    class Inputs : LoggableInputs {
+        var position = Rotation2d()
+        var velocity = Rotation2d()
+        var rollers = Rotation2d()
```

Comment:
Why is this not logged? Also, it should probably be clarified that this is the roller velocity, not angle.

Reply:
💀